README.rst

```diff
@@ -176,6 +176,8 @@ Use the following steps to define a new rule:
    with two files: ``old.yaml`` and ``new.yaml``. The two files represent two versions of the swagger specs that need to be checked for
    backward compatibility.
 
+Pro tip: You can the skeleton generated via ``python -m create_new_rule``.
```

Comment:
What do you think about putting this at the very top, i.e. make it the preferred way of creating a new rule? And rework the description to say where you should adapt / modify the code?

Also: you can _use_ the skeleton...

Reply:
That's definitely a valuable feedback especially because it might simplify a lot the creation of a new rule (read in order and most things are already done) :blush:

I'll wait to complete the work for #22 so I can verify that we're not missing something more  that can be automated.